src/app/server/serverInit.ts

```diff
@@ -110,6 +111,35 @@ export async function serverInit() {
             }
 
             InitSchemas(pool);
+            EmailSvc.sendMail = async (subject: string, message: string, email: string) => {
```

Comment:
Although this matches the example that I have you - I would prefer that it'll be in a separate file as detailed in the article:
https://remult-ts.github.io/guide/using-server-only-packages.html#step-2-implement-the-method

Otherwise, the server init function will be huge

Reply:
Did you give up on this comment?

Reply:
done